cli/src/semgrep/formatter/text.py

```diff
@@ -140,6 +141,9 @@ def _format_lines(
                     # while stripping a string, the ANSI code for resetting color might also get stripped.
                     line = line + colorama.Style.RESET_ALL
 
+            if show_path:
+                yield f" " * 9 + f"{with_color(Colors.cyan, f'{path}', bold=False)}"
```

Comment:
Why is this 9 and not 8 like everything else lol

Reply:
cause the path is slightly indented, to separate it from the intervening messages